### DISCUSSION TITLE
[Question]: How to install python3 cvc5 base api with `--python-bindings`?

### DISCUSSION THREAD

**Discussion by hagozaebii** [cvc5/cvc5#10255] [Q&A] [answered]
Hello. Recently, I wanted to use the commit version of the cvc5 base API in Python, so I installed cvc5 using `--python-bindings`.
However, the cvc5 library was installed somewhere other than the python path I am currently using. When my native python version is 3.8 and the python version actually used through `pyenv` is 3.10, then cvc5 is installed in the 3.8 path. I want to solve this problem, how can I do it?

**Comment by daniel-larraz** [comment, marked as answer]
Sorry for the late reply. PR #10693, that we've just merged into main, changes the installation step to meet the expected behavior.

Please let us know if you find any further issues with the installation.
